lib/email/reader.rb

```diff
@@ -0,0 +1,53 @@
+module Email
+  class Reader
+    EMAIL_REGEX = /(.*)--inline.html/
+    PATH_TO_EMAILS = '/emails'.freeze
+
+    # this will need to change, when used as a gem the local dir will be different
+    def self.default
+      new('./build')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.